src/lib.rs

```diff
@@ -6,6 +6,8 @@ extern crate libc;
 #[macro_use]
 pub mod syslog;
 
+/// Macros and functions for working with
```

Comment:
If you're adding the documentation for the module here you don't need to also add it to `src/ioctl.rs` because it results in duplicated documentation:
![duplicated docs](https://user-images.githubusercontent.com/2815944/61705258-20271f80-ad4e-11e9-98c3-8d768ac4b355.png)

We should decide if we want the module documentation to be placed in src/lib.rs or in the file where the module is defined.

Reply:
It would be better to put the module doc into the corresponding source file.

Reply:
I also prefer that option because for some modules you might have extended documentation and it's making it easier to read the code.